src/project-css/config/database.php

```diff
@@ -62,7 +68,7 @@
             'charset' => 'utf8mb4',
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
-            'strict' => true,
+            'strict' => false,
```

Comment:
What does this do?  Wanting to understand the benefits of true or false.

Reply:
See https://mattstauffer.co/blog/strict-mode-and-other-mysql-customizations-in-laravel-5-2/ 

It enables specific mysql modes that are causing the app to fail during tests. I believe this is related to how Ajay is defining datatypes for fields if it enabled we get Incorrect integer value during tests.